media/static-bundles.json

```diff
@@ -990,6 +996,20 @@
       ],
       "name": "firefox_whatsnew_87_eu_vpn"
     },
+    {
+      "files": [
+        "js/firefox/whatsnew/up-to-date.js",
+        "js/firefox/whatsnew/whatsnew-88-en.js"
+      ],
+      "name": "firefox_whatsnew_88_en"
+    },
+    {
+      "files": [
+        "js/libs/mozilla-traffic-cop.js",
+        "js/firefox/whatsnew/whatsnew-88-exp.js"
+      ],
+      "name": "firefox_whatsnew_88_exp"
```

Comment:
Should it be `firefox_whatsnew_88_exp` or `firefox_whatsnew_88_en_exp`?

Reply:
I don't think it really matters, as long as it's unique. You can add `en` if you want to be specific about which page it's for.